x-pack/plugins/synthetics/server/feature.ts

```diff
@@ -19,6 +24,25 @@ const UPTIME_RULE_TYPES = [
   'xpack.uptime.alerts.monitorStatus',
   'xpack.uptime.alerts.durationAnomaly',
 ];
+
+const disablePublicLocationsPrivilege: SubFeaturePrivilegeGroupConfig = {
+  groupType: 'independent' as SubFeaturePrivilegeGroupType,
+  privileges: [
+    {
+      id: 'enable_public_locations',
+      name: i18n.translate('xpack.synthetics.features.enablePublicLocations', {
+        defaultMessage: 'Enable public locations',
```

Comment:
Since it's under section **Privileges**, "Enable public locations" gives the impression whether user can enable public locations. "Public locations enabled" might be a better fit, not a big deal though.